HikVisionSource/src/main/java/io/vantiq/extsrc/HikVisionSource/HCNetSDK.java

```diff
@@ -0,0 +1,5180 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/*
+ * HCNetSDK.java
+ *
+ * Created on 2009-9-14, 19:31:34
+ */
+
+/**
+ *
+ * @author Xubinfeng
+ */
+
+package io.vantiq.extsrc.HikVisionSource;
+
+import java.io.Serializable;
+
+import com.sun.jna.Callback;
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.NativeLong;
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.Union;
+import com.sun.jna.examples.win32.GDI32.RECT;
+import com.sun.jna.examples.win32.W32API;
+import com.sun.jna.examples.win32.W32API.HANDLE;
+import com.sun.jna.examples.win32.W32API.HDC;
+import com.sun.jna.examples.win32.W32API.HWND;
+import com.sun.jna.ptr.ByteByReference;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.ptr.NativeLongByReference;
+import com.sun.jna.ptr.ShortByReference;
+
+//SDK HCNetSDK.dll
+public interface HCNetSDK extends Library {
+
+	HCNetSDK INSTANCE = (HCNetSDK) Native.loadLibrary("hcnetsdk", HCNetSDK.class);
+
+	public static final int MAX_NAMELEN = 16;
+	public static final int MAX_RIGHT = 32;
+	public static final int NAME_LEN = 32;
+	public static final int PASSWD_LEN = 16;
+	public static final int SERIALNO_LEN = 48;
+	public static final int MACADDR_LEN = 6;
+	public static final int MAX_ETHERNET = 2;
+	public static final int PATHNAME_LEN = 128;
+	public static final int MAX_TIMESEGMENT_V30 = 8;
```

Comment:
Need to either remove all the ???'s or replace them with them with the original comments (language doesn't matter).  Find the originals and run them through google translate.  It would probalby be helpful.

Reply:
i removed the ??? , i guess it would be safer to use the sdk documentation , perhaps i can ask HikVision to provide an up to date version of the file ( which i has to add stuff to it ) with english....

Reply:
Yes, let's pursue that as part of our discussions with them